src/backend/InvenTree/generic/states/fields.py

```diff
@@ -73,6 +77,7 @@ def get_field_info(self, field, field_info):
         return field_info
 
 
+@extend_schema_field(OpenApiTypes.INT)
```

Comment:
This line removes all the choice descriptions; looking into if we can address this in schema generation